src/gui/editorwidgets/qgsrelationwidgetwrapper.h

```diff
@@ -97,14 +111,29 @@ class GUI_EXPORT QgsRelationWidgetWrapper : public QgsWidgetWrapper
     /**
      * Defines the buttons which are shown
      * \since QGIS 3.16
+     * \deprecated since QGIS 3.18
      */
-    void setVisibleButtons( const QgsAttributeEditorRelation::Buttons &buttons );
+    Q_DECL_DEPRECATED void setVisibleButtons( const QgsAttributeEditorRelation::Buttons &buttons );
```

Comment:
```suggestion
    Q_DECL_DEPRECATED void setVisibleButtons( const QgsAttributeEditorRelation::Buttons &buttons ) SIP_DEPRECATED;
```

And please note in the \deprecated dox what the replacement approach is